custom_components/bosch_alarm/translations/en.json

```diff
@@ -25,8 +28,9 @@
         "step": {
             "init": {
                 "data": {
-                    "code": "Arming Code"
-                }
+                    "code": "Arming code"
+                },
+                "description": "Specify an arming code if you wish to require a code for arming and disarming.\nThis code does not need to be related to any codes already set on your panel."
```

Comment:
"code": "The arming code is used to protect the arm/disarm operations locally within Home Assistant. It does not have to match any of the codes configured in the panel."